src/openrct2/windows/ride_list.c

```diff
@@ -411,7 +421,13 @@ static void window_ride_list_scrollmousedown(rct_window *w, int scrollIndex, int
 		return;
 
 	// Open ride window
-	window_ride_main_open(w->list_item_positions[index]);
```

Comment:
Something I would like to encourage is stashing as much as possible in locals, particularly if it reduces the same expression multiple times. So in this case you can do:
`uint8 rideIndex = w->list_item_positions[index];`

And use `rideIndex` below instead of `w->list_item_positions[index]`